tutorial/02 - Styling and Theming.ipynb

```diff
@@ -31,7 +31,7 @@
     "Collapsed": "false"
    },
    "source": [
-    "In this chapter we will learn how to configure various visual aspects of our plots, and how to find out more about what an be configured. "
+    "In this chapter,we will learn how to configure various visual aspects of our plots, and how to find out more about what can be configured. "
```

Comment:
missing a space after the comma, though my personal preference is just not to have a comma there at all.